documentation/server/eventservice.md

```diff
@@ -93,7 +93,7 @@ export class MyComponent {
 
 ## Side Notes
 
-The eventservice has also methods for retrieve events. But if you using the decorators, there is no need to use this. But for the sake of completeness.
+The eventservice also has methods for retrieving events. But if you're using the decorators, there is no need to use them. But for the sake of completeness.
```

Comment:
`But for the sake of completeness.` to But for the sake of completeness, they're included.